src/photos.cs

```diff
@@ -1152,4 +1194,266 @@ interface PHLivePhotoEditingOption {
 		NSString ShouldRenderAtPlaybackTime { get; }
 	}
 #endif
+
+	[Mac (10,13, onlyOn64 : true)]
+	[NoiOS][NoTV]
+	[BaseType (typeof (PHAssetCollection))]
+	interface PHProject {
+
+		[Export ("projectExtensionData")]
+		NSData ProjectExtensionData { get; }
+
+		[Field ("PHProjectTypeUndefined")]
+		NSString TypeUndefined { get; }
```

Comment:
Good point on the type.

It looks like an extensible name (`undefined` is unlikely the only valid name) which makes it a bad candidate for a smart enum (even more as it's exposed as a property).